web/config/sync/user.role.authenticated.yml

```diff
@@ -12,3 +12,4 @@ permissions:
   0: 'access content'
   2: 'view own unpublished content'
   4: 'access toolbar'
+  5: 'view the administration theme'
```

Comment:
If we add this permission to authenticated can we then remove it from other roles as well?